controllers/articles.controller.js

```diff
@@ -34,3 +35,21 @@ exports.getArticleIdComments = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+
+  if (!username || !body) {
```

Comment:
I like that you're checking the inputs here to ensure you have the required information, since these fields are mandatory in the database PSQL would reject the promise for you. However as DB operations can be costly in production I do like this check first.